src/main.rs

```
@@ -38,9 +38,52 @@ async fn load_launch_config(base_path: Option<&PathBuf>) -> errors::Result<Launc
+
+RRDB v{}
+"#,
+        env!("CARGO_PKG_VERSION")
+    )
+}
+
+fn print_banner() {
+    println!("{}", banner());
+}
+
+fn display_base_path(base_path: Option<&PathBuf>) -> String {
```

Clippy (1.96.0)

[warning] function `display_base_path` is never used

(warning)